srcpkgs/cross-arm-none-eabi-gdb/template

```diff
@@ -1,48 +1,36 @@
-# Template file for 'cross-${_triplet}-${_pkgname}'
-_triplet=arm-none-eabi
-_pkgname=gdb
-pkgname=cross-${_triplet}-${_pkgname}
-version=8.3
+# Template file for 'cross-arm-none-eabi-gdb'
+pkgname=cross-arm-none-eabi-gdb
+version=10.2
 revision=1
-wrksrc="${_pkgname}-${version}"
+wrksrc=gdb-${version}
 build_style=gnu-configure
-pycompile_dirs="/usr/share/${_pkgname}"
-configure_args="
- --disable-gdbserver
- --disable-nls
- --disable-werror
- --host=${XBPS_CROSS_TRIPLE}
- --prefix=/usr
- --target=${_triplet}
- --with-babeltrace
- --with-expat
- --with-mpfr
- --with-system-readline
- --with-system-zlib
- --without-isl
- $(vopt_with guile)
- $(vopt_with python)
-"
-hostmakedepends="autoconf automake bison flex pkg-config texinfo"
-makedepends="babeltrace-devel expat-devel mpfr-devel ncurses-devel
- readline-devel zlib-devel $(vopt_if guile guile-devel)
- $(vopt_if python 'python3-devel gettext-devel')"
-short_desc="GNU Debugger"
+configure_args="--target=arm-none-eabi --disable-werror --disable-nls --with-system-readline
+ --with-system-gdbinit=/etc/gdb/gdbinit --with-system-zlib --without-isl
```

Comment:
Why does `avr-gdb` not have `without-isl`?

Reply:
I honestly don't know. I am not sure why this library is needed here.

Reply:
> The cross gdbs are missing gdbserver now

```console
$ ./xbps-src pkg avr-gdb
...
/builddir/gdb-10.2/gdbserver/remote-utils.cc:1235: undefined reference to `using_threads'
/usr/bin/ld: server.o: in function `main':
/builddir/gdb-10.2/gdbserver/server.cc:3826: undefined reference to `initialize_low()'
...
```

I don't think that gdbserver makes sense for avr and arm-none-eabi.

Reply:
Argh, sorry. My mind was in previous revisions of this PR.

Indeed, neither of those packages have gdbserver currently.